web/src/containers/Wallet/MainWallet.js

```diff
@@ -228,14 +226,6 @@ class Wallet extends Component {
 		this.setState({ activeTab });
 	};
 
-	openContactUs = () => {
```

Comment:
@ahslr why did you remove this?

Reply:
The `openContactUs` is passed as a prop to the `assets block` component. Since the logic that triggers a rendering in assets block for that part is removed, I removed the prop as well in the component and stop passing it to the child as well. @abeikverdi

Reply:
Ok. looks like a dead code then.